code/game/area/Space_Station_13_areas.dm

```diff
@@ -104,6 +104,17 @@ NOTE: there are two lists of areas in the end of this file: centcom and station
 	ambientsounds = MAINTENANCE
 	valid_territory = FALSE
 
+/area/maintenance/arena
+	name = "The Arena"
+	icon_state = "maint_bar"
+
+/area/greytopia
+	name = "Greytopia"
+	ambientsounds = MAINTENANCE
+	icon_state = "away"
+	requires_power = FALSE
+	has_gravity = TRUE
+	dynamic_lighting = DYNAMIC_LIGHTING_FORCED

```

Comment:
Ok so there's an area on the station that just breaks the laws of physics?

Reply:
Most ruins do but if its that objectionable (these are hidden, dont show up every round) ill ghettoize it